server/cmwell-cons/src/main/scala/ComponentConf.scala

```diff
@@ -295,6 +296,12 @@ case class CassandraConf(home: String,
   }
 }
 
+object CassandraConf{
+  def checksum  =
+    UtilCommands.verifyComponentConfNotChanged("apache-cassandra", "conf/cassandra.yaml", "13EDA21C959FE5985A17385A64DE5817")
```

Comment:
add comment telling what version is the one that is hashed

Reply:
done